app/views/courses/_basic_details_tab.html.erb

```diff
@@ -31,6 +31,18 @@
         <dd class="govuk-summary-list__actions"></dd>
       </div>
 
+      <% unless course.level == 'further_education' %>
```

Comment:
Suggest doing this with edit options when we have them:
`if @course.meta[‘edit_options’][‘age_range_in_years’].any?`

But this is fine for now.